tests/Integration/Behaviour/Features/Context/SharedStorage.php

```diff
@@ -77,4 +77,14 @@ public function set($key, $resource)
     {
         $this->storage[$key] = $resource;
     }
+
+    /**
+     * @param string $key
+     */
+    public function clear($key)
+    {
+        if (isset($this->storage[$key])) {
```

Comment:
array_key_exists instead? Maybe we'll want to keep null values. Wdyt?

Reply:
Hmm, not sure if it's worth covering case that does not exist and. :thinking:

Reply:
Errr ... yeah @PierreRambaud the storage is intended to keep track of items created in our behat tests. Why would be keep a `null` value ? 🤔

Reply:
Because you wanted to tried with an empty data :p